taverna-language-commandline/src/main/java/org/apache/taverna/commandline/CommandLineTool.java

```diff
@@ -0,0 +1,252 @@
+package org.apache.taverna.commandline;
```

Comment:
Could you set the package name something like
`org.apache.taverna.language.commandline` 
? To avoid confusion with the executeworkflow-commandline, which we are keeping outside for now.